README.md

```diff
@@ -1,6 +1,21 @@
-# TinyButStrong template engine
+TinyButStrong template engine
+-----------------------------
 
 TBS is a PHP template engine for pro and beginners.
 Only 1 class with few methods and properties, but it can do may things for any text templates, including HTML and XML.
 The only engine that enables W3C compliant templates.
 It has many plugins including OpenTBS.
+
+
+Installation
+------------
+
+The preferred way to install this extension is through [composer](http://getcomposer.org/download/).
+
+Either run
+
+```
+php composer.phar require skrol29/tinybutstrong "*"
```

Comment:
this should be just `composer require skrol29/tinybutstrong`

Reply:
fair remark